src/components/Missions.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import { createJoin } from '../redux/missions/missions';
+
+const Missions = (props) => {
+  const prop = props;
+  const dispatch = useDispatch();
+  const { join } = prop.mission;
+  const handleJoin = () => {
+    const toggleJoin = !join;
+    const setJoin = (() => ({ ...prop.mission, join: toggleJoin }));
+    dispatch(createJoin(setJoin()));
+  };
```

Comment:
It is good to use prop here but you already have the join data in your store it would be better to use join property from your mission reducer
![image](https://user-images.githubusercontent.com/74744787/151157235-2a6a9c3f-a57b-4a2e-a651-81b10ee54437.png)